lms/models/application_instance.py

```diff
@@ -38,6 +38,11 @@ class ApplicationInstance(BASE):
         "OAuth2Token", back_populates="application_instance"
     )
 
+    #: A list of all the GroupInfo's for this application instance.
+    group_infos = sa.orm.relationship(
+        "GroupInfo", back_populates="application_instance"
+    )
```

Comment:
We're not actually making use of this but this just means that, given an `ApplicationInstance` model object instance, `my_application_instance.group_infos` will be a list of all its related `GroupInfo`s.